src/main/fc/runtime_config.c

```diff
@@ -23,6 +23,8 @@
 #include "fc/runtime_config.h"
 #include "io/beeper.h"
 
+const char *armingDisableFlagNames[] = { ARMING_DISBALED_FLAG_NAMES };
```

Comment:
It should be wrapped with IFNDEF MINIMAL_CLI

Also DISBALED is missing an A :)